src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -732,7 +732,7 @@ def _generate_tools_meta(
         This is a private interface for vscode extension, so do not change the interface unless necessary.
 
         Usage:
-        from promptflow import PFClient
+        from promptflow.client import PFClient
```

Comment:
I see two changes for this, which one is the recommended way to import PFClient?

`from promptflow.client import PFClient`

or

`from promptflow._sdk._pf_client import PFClient`

Reply:
We suggest `from promptflow.client import PFClient` for public usage, some of the files be modified to 'from promptflow._sdk.xx import xx' because we don't clean imports in core yet, just make import path more short to avoid circular import as possible as we can.